doc/modules/cross_validation.rst

```diff
@@ -405,8 +405,8 @@ for cross-validation against time-based splits.
 
 .. warning::
 
-  Contrary to :class:`StratifiedKFold`, **the ``labels`` of
```

Comment:
According to http://sphinx-doc.org/rest.html#inline-markup presently, ReST cannot have nested inline markups. No other warnings in docs seemed to have any emphasis though. Not sure if there is any other way to do this.